### DISCUSSION TITLE
Ruler tool uses a raw translation key in its name

### DISCUSSION THREAD

**Discussion by TheMightyDarkness** [Mojang/minecraft-editor#191] [Bugs]
**The Bug:**

The ruler tool name in the Bedrock Editor uses a raw translation key named resourcePack.editor.toolRail.rulerTool.Title instead of its real name.

**Steps to Reproduce:**

1. Open the Minecraft Bedrock Editor in the latest Preview build.
2. Create a new project. While creating the project, the project should be in Creative mode, on easy difficulty, have the world type infinite world, and cheats on. Do not enable additional behavior packs or resource packs, do not use experiments and do not enable Education Edition.
3. Once the project is created, there will be a sidebar on the left with 6 icons, which is the tool rail. These tools are Selection, Paste Preview, Line, Summon Tool, Brush and Ruler respectively from top to bottom.
4. Hover over the ruler tool. (The ruler tool is the bottom of the 6 tools on the left sidebar, which is the tool rail.)
5. There will be a broken title for the Ruler tool, using its raw translation key.

**Expected Results:**

The ruler tool will have a real title named the Ruler Tool.

**Observed Results:**

The ruler tool uses a broken title, using its raw translation key.

**Notes:**
This was copied from the Minecraft bug tracker at Jira: [MCPE-185171](https://bugs.mojang.com/browse/MCPE-185171)

**Frequency:**
This always occurs.

**Affected versions:**
1.21.30.22 Preview, 1.21.30.23 Preview, 1.21.50.24 Preview

**Player mode:**
Single-Player mode

**Comment by bryhaw** [comment]
Great catch! I'll log this one internally. Thanks!
